frontend/src/components/ViewTrips/trip.js

```diff
@@ -2,31 +2,38 @@ import React from 'react';
 
 import Button from 'react-bootstrap/Button';
 
-import { getUserEmailFromUid } from '../Utils/temp-auth-utils.js'
+import { timestampToISOString } from '../Utils/time.js';
+import { getUserEmailFromUid } from '../Utils/temp-auth-utils.js';
 import ViewActivitiesButton from './view-activities-button.js';
 
 /**
  * Returns the date range of the trip associated with the Trip document data
- * `tripObj`.
+ * `tripData`.
  *
  * Notes:
  *  - tripObj will always contain valid start_date and end_date fields.
  *  - When the Firestore Timestamps contained in `tripObj` converted to js
  *    dates, the months are 0 indexed rather than 1 indexed so they must be
  *    incremented by 1 in order for the month to be correct.
  *
- * @param {firebase.firestore.DocumentData} tripObj Object containing the fields
+ * @param {firebase.firestore.DocumentData} tripData Object containing the fields
  *    and values for a Trip document.
  * @return Date range of the trip (if it exists).
  */
-export function getDateRange(tripObj) {
-  const startDate = tripObj.start_date.toDate();
-  const endDate = tripObj.end_date.toDate();
+export function getDateRange(tripData) {
+  const startDate = tripData.start_date.toDate();
+  const endDate = tripData.end_date.toDate();
   return `${startDate.getMonth() + 1}/${startDate.getDate()}/`  +
       `${startDate.getFullYear()} - ${endDate.getMonth() + 1}/` +
       `${endDate.getDate()}/${endDate.getFullYear()}`;
```

Comment:
not sure if I mentioned this on this PR or the other one, but this will always get these fields relative to the client's local time. beware.

Reply:
Added #89.